pkg/scheduler/algorithm/predicates/metadata_test.go

```diff
@@ -1635,7 +1635,7 @@ func TestPodSpreadCache_removePod(t *testing.T) {
 			} else {
 				deletedPod = tt.deletedPod
 			}
-			podSpreadCache.removePod(deletedPod, tt.preemptor, tt.nodes[tt.nodeIdx])
+			podSpreadCache.updatePod(deletedPod, tt.preemptor, tt.nodes[tt.nodeIdx], -1)
```

Comment:
not a big fan of the numbers. Maybe just add `addPod` and `removePod` that call `updatePod`? But this is a nit.

Reply:
Updated.